### PR TITLE
Feature/wketchum trigger decoding updates

### PR DESCRIPTION
**Pull request by wesketchum**
This is a start to this, but doesn't yet do the same thing for the 'v3' reader module.

### CODE REVIEW

**Comment by wesketchum** [comment]
Also, needs [dunecore#111 ](https://github.com/DUNE/dunecore/pull/111).

File: duneprototypes/Protodune/hd/RawDecoding/PDHDTriggerReader_module.cc [hunk 1]
```diff
@@ -17,6 +17,7 @@
 #include "dunecore/DuneObj/RDStatus.h"
 #include "dunecore/DuneObj/DUNEHDF5FileInfo2.h"
 #include "dunecore/HDF5Utils/HDF5RawFile2Service.h"
+#include "daqdataformats/v4_4_0/Fragment.hpp"
```
Comment (tomjunk) [inline duneprototypes/Protodune/hd/RawDecoding/PDHDTriggerReader_module.cc:20]: Be careful here -- including the v4_4_0 fragment with DUNEHDF5FileInfo2 mixes old and new formats.  You have included the daqdataformats/v3_4_1/Fragment.hpp and the daqdataformats/v4_4_0/Fragment.hpp headers -- I'm a bit surprised this compiles.

**Comment by tomjunk** [comment]
And this is a breaking change in that it produces different stuff from the old decoder.  It may be okay to break it because no one was depending on the old way of doing things, but in the future, when we have many collaborators analyzing data, we should strive for backwards compatibility.

**Comment by calcuttj** [comment]
Hey Wes, any updates on this?

**Comment by tomjunk** [comment]
I think I can fix the include myself approve it after our conversation yesterday.

**Pull request merged**